### PR TITLE
dark mode and color bubbles

### PR DESCRIPTION
**Pull request by pvinis**
for dark mode, i just inverted the image. for some reason it shows it bigger, but didnt have the time to check why. changes and recommendations are welcome. :)
this is for issue #1.

### CODE REVIEW

**Comment by revett** [comment]
Thanks @pvinis :+1:

**Comment by nickgartmann** [comment]
Here are some images files that will do the trick. The key to sizing the icons is DPI, your inverted image was 72 DPI where the rest of them are 326.

docker-dark 2x

docker-alt-dark 2x

I added a dark-alt image to make it a bit easier for colorblind users to tell the difference.

File: Boot2Docker Status/AppDelegate.m [hunk 1]
```diff
@@ -30,6 +31,12 @@ - (void)applicationDidFinishLaunching:(NSNotification *)aNotification {
     
     _standardUserDefaults = [NSUserDefaults standardUserDefaults];
     
+	_darkMode = NO;
+    NSString *osxMode = [_standardUserDefaults stringForKey:@"AppleInterfaceStyle"];
```
Comment (nickgartmann) [inline Boot2Docker Status/AppDelegate.m]: Please put this into the `renderIcon` method as well, so the icon will respond to changes from light mode to dark mode and vice versa (likely only affects flux users).

File: Boot2Docker Status/AppDelegate.m [hunk 2]
```diff
@@ -172,11 +180,24 @@ - (void)stopBoot2Docker {
...
-        _statusItem.image = [NSImage imageNamed:@"docker-alt"];
+		_statusItem.image = [self _iconWithNotificationBubbleWithColor:[NSColor redColor]];
     }];
     
     [task launch];
 }
 
+- (NSImage*)_iconWithNotificationBubbleWithColor:(NSColor*)color {
+	NSImage* image = [(_darkMode ? [NSImage imageNamed:@"docker-dark"] : [NSImage imageNamed:@"docker"]) copy];
+
```
Comment (nickgartmann) [inline Boot2Docker Status/AppDelegate.m:221]: Add this to use the alternative images when boot2docker is down (for colorblind users)

``` objectivec
if(![self isBoot2DockerRunning]) {
        image = [(_darkMode ? [NSImage imageNamed:@"docker-alt-dark"] : [NSImage imageNamed:@"docker-alt"]) copy];
}
```

Comment (pvinis) [inline Boot2Docker Status/AppDelegate.m:221]: why do we need the alt image? when boot2docker is down we just put a red dot, right?

**Comment by nickgartmann** [comment]
Thanks for the PR @pvinis! Really like the way the inverted looks in dark mode, and I love the change to manually draw the status indicators on the image.

A few small changes to be made before I'd want to merge everything in (I can do them myself if you prefer, but I figured I'd give you the option to have the full-feature worth of commits in the repo)

File: Boot2Docker Status/AppDelegate.m (hunk 2 above)
Comment (nickgartmann) [inline Boot2Docker Status/AppDelegate.m:221]: it gives a non-color-based indication of the state, red-green colorblind users will not be able to differentiate between the states without it.

**Comment by pvinis** [comment]
@nickgartmann :ok_hand: now it uses the 'alt' icons for everything except green. and checks for dark mode everytime.

**Comment by nickgartmann** [comment]
:+1: nice work! merging and bumping to version 1.1

**Pull request merged**